### DISCUSSION TITLE
Overlapping entities

### DISCUSSION THREAD

**Discussion by Zim1-finest** [explosion/spaCy#9619] [Help: Model Advice] [answered]
I have a basic beginner question: Is it possible to have overlapping entities on one spaCy model?

For example if I create a new entity class called food and train spaCy to extract food items can I also then create another entity named Pasta where I train the model to extract only pasta entities or will I need a new model to train the overlapping entities? 

FYI, I am using food here only as an example. In the real case I want to use locations and I am thinking whether to add the new location lables to the pretrained model or start with an empty spaCy model.

**Comment by polm** [comment, marked as answer]
You can't do this with an NER model because each token can only have one NER label. You can do this with [spancat](https://spacy.io/api/spancategorizer). 

You can also train multiple models and have one overwrite the other, see the [double NER project](https://github.com/explosion/projects/tree/v3/tutorials/ner_double).

**Reply by Zim1-finest** [reply]
I just went through the spancat and the double NER. It seems like that is exactly what I needed. 

Thank you...
